#################
# Object Search #
#################

#First orientation (pan resp tilt in degrees) that ptu moves to during initial spiral search.
starting_orientation_pan: -18
starting_orientation_tilt: -24

#how many seconds the object recognition is given between pan/tilt adjustments
searching_timer: 5

#Size of buckets for ism recognition in unit [m] 
sensitivity: 0.15

#Name of file with tables containing both models and recordings of scenes to be recognized.
#dbfilename: /data/asr_recordings/rotation_recognition/record.sqlite.scene_relative_rotation
dbfilename: /home/SMBAD/braun/data/record.sqlite.scene_reel

#Prevent points and arrows in vote visualization from being erased after views containing them have been searched.  
keepUnfoundObjectVotesViz: false

#Score of a view under which we abort searching. Here: No reward and staying at your position.
abortionScore: 0.5

#ISM hierarchy displacement for its visualization (including direction in z-direction of used coordinate frame).
step: -0.1

#Confidence of a recognized scene that suffices to end search.
endConfidence: 1

#size of each quad of the sphere in degrees
quad_pan_deg: 5
quad_tilt_deg: 5

#how much % of the actual FoV the PTU is assumed to see
window_adjustment: 0.5

#the weights for reward, cost and exploration
#R is typically 1, C in [0.5, 2] and E in {0, 1}
alpha_R: 0.5
alpha_C: 0.5
#alpha_E: 1

#THIS IS ALL EVALUATION STUFF

#For evaluation purposes
enableISM: true

#Measure times and data output.
enableEvaluation: true

#Absolute path to script that saves marker messages.
markerSaverScript: /home/SMBAD/braun/svn/trunk/perception/scene_understanding/active_scene_recognition_psm/launch/record_visualisation

#basepath to store the evaluation data
basePath: /home/SMBAD/braun/data/eval/psm/

